talisker/metrics.py

```diff
@@ -132,3 +139,175 @@ def inc(self, amount=1, **labels):
             client = talisker.statsd.get_client()
             name = self.get_statsd_name(labels)
             client.incr(name, amount)
+
+
+histogram_archive = 'histogram_archive.db'
+counter_archive = 'counter_archive.db'
+
+
+def histogram_sorter(sample):
+    # sort histogram samples in order of bucket size
+    name, labels, _ = sample
+    return name, float(labels.get('le', 0))
+
+
+def prometheus_cleanup_worker(pid):
+    """Clean up after a multiprocess worker has died."""
+    if prometheus_client is None:
```

Comment:
what's this guarding against? If prometheus_client fails to import it's set to `False`, not `None`

Reply:
It's just defence in depth, but I've fixed the check